searchcore/src/vespa/searchcore/proton/server/documentdb.cpp

```diff
@@ -205,9 +206,12 @@ DocumentDB::DocumentDB(const vespalib::string &baseDir,
     _clusterStateHandler.addClusterStateChangedHandler(this);
     // Forward changes of cluster state to bucket handler
     _clusterStateHandler.addClusterStateChangedHandler(&_bucketHandler);
-    for (auto subDb : _subDBs) {
-        _lidSpaceCompactionHandlers.push_back(std::make_unique<LidSpaceCompactionHandler>(*subDb, _docTypeName.getName()));
-    }
+
+    // Lid space compaction handlers are added in the same order as sub dbs are defined in DocumentSubDbCollection.
```

Comment:
Is ordering important ? If so this is brittle.

Reply:
Ordering is not relevant, comment is removed.